src/dndbeyond/content-scripts/character.js

```diff
@@ -223,6 +223,12 @@ function rollItem(force_display = false) {
             damages.push(String(rage_damage));
             damage_types.push("Rage");
         }
+        if (character.hasClassFeature("Rage") && character.getSetting("barbarian-rage", false) && character.hasClassFeature("Divine Fury") &&
+            properties["Attack Type"] == "Melee") {
+            const barbarian_level = character.getClassLevel("Barbarian");
+            damages.push(String(`1d6+${Math.floor(barbarian_level / 2)}`));
```

Comment:
Using ``` `${}` ``` already makes it into a string, so you don't need to add a call to `String()`

Reply:
I think that was a holdover from how I was doing it at first that I didn't clean up... Standby...